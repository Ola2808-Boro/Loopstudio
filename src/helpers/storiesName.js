export const stories = {
  atoms: "Components/atoms/",
  molecules: "Components/molecules/",
  organisms: "Components/organisms/",
  temapltes: "Components/temapltes/",
};
