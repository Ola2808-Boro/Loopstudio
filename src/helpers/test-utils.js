import React from "react";
import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import { theme } from "assets/styles/theme";
import GlobalStyle from "assets/styles/GlobalStyle";
import DataProvider from "providers/DataProvider";
const AllTheProviders = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      <DataProvider>
        <GlobalStyle />
        {children}
      </DataProvider>
    </ThemeProvider>
  );
};

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options });

// re-export everything
export * from "@testing-library/react";

// override render method
export { customRender as render };
