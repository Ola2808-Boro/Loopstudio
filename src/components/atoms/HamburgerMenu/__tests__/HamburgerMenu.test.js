import Menu from "../HamburgerMenu";
import { render, fireEvent } from "helpers/test-utils";
import renderer from "react-test-renderer";

describe("Test button component", () => {
  it("default style", () => {
    const tree = renderer.create(<Menu />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
