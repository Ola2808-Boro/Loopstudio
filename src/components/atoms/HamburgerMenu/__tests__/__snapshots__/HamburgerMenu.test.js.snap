// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test button component default style 1`] = `
<svg
  className="sc-bczRLJ ccfSbU"
>
  icon-hamburger.svg
</svg>
`;
